cmd/vaults/commands.go

```diff
@@ -32,8 +32,8 @@ import (
 var vaultNameRx = regexp.MustCompile(`^([a-zA-Z_][a-zA-Z0-9_]*)[.]([a-zA-Z_][a-zA-Z0-9_]*$)`)
 
```

Comment:
lots of changes in this file, but in essence:
- moved the flags from `create` to `stream`
- we're now passing multiple tables and schemas to the database manager